components/raftstore/src/store/peer.rs

```diff
@@ -3965,10 +3963,24 @@ mod tests {
                 renew_lease_time,
                 must_pass_epoch_check: false,
             });
+        };
+        for index in 1..=100 {
+            push_proposal(&mut pq, index);
         }
         let mut pre_remove = 0;
-        for remove_i in &[0, 1, 65, 98, 100] {
-            let remove_i = *remove_i;
+        for remove_i in 1..=100 {
```

Comment:
How long does the case take?

Reply:
There is no time-consuming operation, the test will finish before a human can notice.